65black/StackOlite
# #160201253 Post A Question

#### What does this PR do?
This PR consumes the `/api/v1/questions/` endpoint from [this repo](https://github.com/65black/StackOverflow-lite).

#### How should this be manually tested?
 - Clone this repo and open index.html with Chrome.
 - Log in to an existing account.
 - Navigate to the ask question page.
 - Fill in the form and click on 'ask'.
 - You should be redirected to the questions page with your newly added question at the top. 

#### Screenshots
![ask](https://user-images.githubusercontent.com/36507972/45908476-b455ec80-bdf4-11e8-8e77-5e523675d514.JPG)

#### What are the relevant pivotal tracker stories?
#160201253
